packages/core/src/utils.ts

```diff
@@ -714,3 +715,55 @@ function toPreciseInteger(a: number, precision?: number): number {
   }
   return Math.round(a * Math.pow(10, precision));
 }
+
+/**
+ * Returns a human-readable string for a FHIR Range datatype, taking into account comparators and one-sided ranges
+ * @param range A FHIR Range element
+ * @returns A human-readable string representation of the Range
+ */
+export function formatRangeString(range: Range | undefined): string {
+  if (!range || (!range.low && !range.high)) {
+    return '';
+  }
+
+  if (range.low && !range.high) {
+    return `>= ${formatQuantityString(range.low)}`;
+  }
+
+  if (!range.low && range.high) {
+    return `<= ${formatQuantityString(range.high)}`;
+  }
+
+  return `${formatQuantityString(range.low)} - ${formatQuantityString(range.high)}`;
+}
+
+/**
+ * Returns a human-readable string for a FHIR Quantity datatype, taking into account units and comparators
+ * @param quantity A FHIR Quantity element
+ * @returns A human-readable string representation of the Quantity
+ */
+export function formatQuantityString(quantity: Quantity | undefined): string {
```

Comment:
Reconcile this with `QuantityDisplay`: https://github.com/medplum/medplum/blob/main/packages/react/src/QuantityDisplay/QuantityDisplay.tsx

Reply:
Same as above. Thought I had _moved_ the code, but think it got duplicated in the merge. Will resolve and test